app/sidekiq/decision_review/failure_notification_email_job.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module DecisionReview
+  class FailureNotificationEmailJob
+    include Sidekiq::Job
+
+    sidekiq_options retry: false, unique_for: 30.minutes
+
+    SAVED_CLAIM_MODEL_TYPES = %w[
+      SavedClaim::NoticeOfDisagreement
+      SavedClaim::HigherLevelReview
+      SavedClaim::SupplementalClaim
+    ].freeze
+
+    TEMPLATE_IDS = {
+      'HLR' => Settings.vanotify.services.benefits_decision_review.template_id.higher_level_review_form_error_email,
+      'NOD' => Settings.vanotify.services.benefits_decision_review.template_id.notice_of_disagreement_form_error_email,
+      'SC' => Settings.vanotify.services.benefits_decision_review.template_id.supplemental_claim_form_error_email
+    }.freeze
+
+    ERROR_STATUS = 'error'
+
+    STATSD_KEY_PREFIX = 'worker.decision_review.failure_notification_email'
+
+    def perform
+      return unless enabled? && (submissions.present? || submission_uploads.present?)
+
+      send_form_emails
+      send_evidence_emails
+
+      nil
+    end
+
+    private
+
+    def vanotify_service
+      @service ||= ::VaNotify::Service.new(Settings.vanotify.services.benefits_decision_review.api_key)
+    end
+
+    # Fetches SavedClaim records for DecisionReview that have an error status for the form or any evidence attachments
+    def errored_saved_claims
+      @errored_saved_claims ||= ::SavedClaim.where(type: SAVED_CLAIM_MODEL_TYPES)
+                                            .where(delete_date: nil)
+                                            .where('metadata LIKE ?', '%error%')
+                                            .order(id: :asc)
+    end
+
+    def submissions
+      @submissions ||= begin
+        guids = errored_saved_claims.select { |sc| JSON.parse(sc.metadata)['status'] == ERROR_STATUS }.pluck(:guid)
+        ::AppealSubmission.where(submitted_appeal_uuid: guids).failure_not_sent
+      end
+    end
+
+    def submission_uploads
+      @submission_uploads ||= begin
+        uploads = errored_saved_claims.map { |sc| JSON.parse(sc.metadata)['uploads'] }
+        ids = uploads.flatten.select { |upload| upload&.fetch('status') == ERROR_STATUS }.pluck('id')
+
+        ::AppealSubmissionUpload.where(lighthouse_upload_id: ids).failure_not_sent
+      end
```

Comment:
Is this going to catch evidence uploads where the claim was successful but the evidence failed?